### PR TITLE
Run under jenkins CI (4-2-stable)

### PR DESCRIPTION
**Pull request by d-w-moore**

### CODE REVIEW

**Comment by trel** [comment]
if you're happy with the [34] issue.... please add the # and we'll get it merged - same for master.

**Comment by d-w-moore** [comment]
Doing some more clean-up and getting Ubuntu 16.04 taken care of too... it was missing.  Meanwhile we should probably merge #32 and #33

**Comment by d-w-moore** [comment]
Squashed and #'d now. These appear ready for merge.

File: jenkins_testing/Dockerfile.ubuntu_18.client_runner [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+RUN apt install -y postgresql
+RUN apt install -y nginx
+RUN apt install -y curl
+RUN apt install -y python-pycurl
+RUN apt install -y rsyslog
+
+COPY ub_config_and_test_rest_clients.sh configure_and_start_iRODS.sh configure_and_start_NGINX.sh /tmp/
+RUN chmod +x    /tmp/ub_config_and_test_rest_clients.sh \
+                /tmp/ub_config_and_test_rest_clients.sh \
+                /tmp/configure_and_start_iRODS.sh
```
Comment (trel) [inline jenkins_testing/Dockerfile.ubuntu_18.client_runner:29]: duplicate here, but can fix in another commit

File: jenkins_testing/PKGFILE_install_dependencies [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    command grep -Ev '^irods-(dev|icommands|runtime|server|database)' |\
+    remove_duplicates
+}
+
+centos_setup() 
+{
+    yum -y install epel-release sudo wget
+    sudo rpm --import https://packages.irods.org/irods-signing-key.asc
+    wget -qO - https://packages.irods.org/renci-irods.yum.repo |\
+     sudo tee /etc/yum.repos.d/renci-irods.yum.repo
```
Comment (trel) [inline jenkins_testing/PKGFILE_install_dependencies:28]: whitespace

File: jenkins_testing/Dockerfile.ubuntu_18.client_runner (hunk 1 above)
Comment (d-w-moore) [reply jenkins_testing/Dockerfile.ubuntu_18.client_runner:29]: Should I add the de-duplicating commit as part of this pull request, even if it's closed?

Comment (trel) [reply jenkins_testing/Dockerfile.ubuntu_18.client_runner:29]: no, it's already merged - new PR

Comment (trel) [reply jenkins_testing/Dockerfile.ubuntu_18.client_runner:29]: should it be a third script?   rather than just removal of a duplicate?

**Pull request merged**